docs/create_development_environment.md

```diff
@@ -73,7 +73,7 @@ module.exports = {
 }
 ```
 
-The `entry` field is the entery point of our project. The plugin we use instruct webpack to add a banner line at the begining of the generated code that will contains the shabeng syntax required by RedisGears along side the library name.
+The `entry` field is the entry point of our project. The plugin we use instruct webpack to add a banner line at the begining of the generated code that will contains the shebang syntax required by RedisGears along side the library name.
```

Comment:
The `entry` field represents the entrypoint for our project. The plugin we use instructs webpack to add a banner line at the beginning of the generated code. The generated code will be prefixed by the shebang syntax required by RedisGears alongside the library name.